README.md

```diff
@@ -40,6 +51,8 @@ $ imap-data-access query --start-date 20240101 --end-date 20241231 --output-form
 
 Download a level 0 SWE file on 2024/01/05
 
+Note: Make sure the directory configuration is correct, see Data Directory section for more information.
```

Comment:
You can add a warning/tip admonition here. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Also, if you are indicating a user should look at a section, it is helpful to link out to that section directly.
```suggestion
> [!NOTE]
> Make sure the directory configuration is correct, see the [Data Directory section](https://github.com/IMAP-Science-Operations-Center/imap-data-access#data-directory) for more information.
```